paasta_tools/setup_kubernetes_cr.py

```diff
@@ -64,6 +67,13 @@
 }
 
 
+INSTANCE_TYPE_TO_NAMESPACE_LOADER = {
```

Comment:
this is maybe a bit of a misleading name since this isn't actually a loader/getter/etc - it's actually a mapping